build/pkgs/conway_polynomials/SPKG.rst

```diff
@@ -1,12 +1,26 @@
-conway_polynomials: Tables of Conway polynomials over finite fields
+Python interface to Frank Lübeck's Conway polynomial database
```

Comment:
Need to keep `conway_polynomials:` at the beginning - that's the general format (for the TOC in the package list)

Reply:
By the way, `./sage -package create conway_polynomials --pypi --type standard` sets up these things automatically in the right format

Reply:
The `dependencies` file also needs updating

Reply:
Both should be fixed.